frontend/device/input.lua

```diff
@@ -38,6 +38,15 @@ local ABS_MT_POSITION_X = 53
 local ABS_MT_POSITION_Y = 54
 local ABS_MT_TRACKING_ID = 57
 local ABS_MT_PRESSURE = 58
+
+-- For device orientation events (ABS.code)
+
+local ABS_PRESSURE = 24
```

Comment:
+1, and also cannot quite understand why they restrict the usage of page up / down button on voyage.